sig-cli/README.md

```diff
@@ -25,6 +25,16 @@ Covers kubectl and related tools. We focus on the development and standardizatio
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-cli)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fcli)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **common**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/common/master/OWNERS
+- **kubectl**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kubectl/master/OWNERS
+
```

Comment:
Kubectl's main location is in the kubernetes repo:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/OWNERS